src/helpers/links.js

```diff
@@ -52,16 +52,18 @@ import {
 } from './consts';
 
 export const landingPageSectionLink = (landingPage, filterValues) => {
-  const linkParts = [SEARCH_GRAMMAR_LANDING_PAGES_SECTIONS_LINK[landingPage]];
+  const linkParts = (SEARCH_GRAMMAR_LANDING_PAGES_SECTIONS_LINK[landingPage] || '').split('?');
```

Comment:
Why not use 
```new URL`(...)``` to parse the flags?
https://developer.mozilla.org/en-US/docs/Web/API/URL/URL